src/ServiceProvider.php

```diff
@@ -5,6 +5,9 @@
 use Statamic\Providers\AddonServiceProvider;
 
 class ServiceProvider extends AddonServiceProvider {
+
+  protected $publishAfterInstall = false;
```

Comment:
Hey, just wondering why you have added this line, I think publishing after install is desired is it not?

Reply:
Oh, I added it since you had the instructions in the README for manually publishing them. But yes, it's not needed.

Reply:
Ya. They were added to the readme before Statamic auto published on install. I have removed this step from the install part of the readme. Thanks.